File: 프로그래머스/0/120812. 최빈값 구하기/최빈값 구하기.py
def solution(array):
    cnt = [0] * 1001
    
    for i in array:
        cnt[i] += 1
    
    n = 0
    for c in cnt:
        if c == max(cnt):
            n += 1
            
    if n > 1:
        return -1
    else:
        return cnt.index(max(cnt))

File: 프로그래머스/0/120812. 최빈값 구하기/최빈값 구하기.js
function solution(array) {
    let cnt = new Array(1001).fill(0);
    
    for (let i of array) {
        cnt[i]++;
    }

    let n = 0;
    for (let c of cnt) {
        if (c == Math.max(...cnt)) {
            n++;
        }
    }

    if (n > 1) {
        return -1;
    } else {
        return cnt.indexOf(Math.max(...cnt));
    }
}